index.bs

```diff
@@ -800,6 +809,14 @@ authorizing an authenticator with which to complete the operation.
         omitted, the RP ID will be set to the [=origin=] specified by the {{WebAuthentication}} object's [=relevant settings
         object=].
 
+    - The <dfn>attachment</dfn> parameter contains authenticator attachment descriptions, which are used as an additional
+        constraint on which authenticators are eligible to participate in a [[#makeCredential]] or [[#getAssertion]] operation.
+        See [[#attachment]] for a description of the attachment values and their meanings.
+
+    - The <dfn>requireResidentKey</dfn> parameter describes the [=[RPS]=]' requirements regarding availability of the 
+        <a>credential private key</a>. If the parameter is set to <i>true</i>, the authenticator SHOULD create a <a>resident
+        key</a> when creating a <a>scoped credential</a>. Similiar to {{attachment}}, the {{requireResidentKey}} parameter is an additional constraint on which authenticators are eligible for a [[#makeCredential]] or [[#getAssertion]] operation.
```

Comment:
Please use the [=term=] style of markup instead of &lt;a>term&lt;/a> to be consistent with the rest of the spec.